From fdb8bc6e3b743ea11e1b809286c2b06808f57a5d Mon Sep 17 00:00:00 2001
From: Andrew Lenart <w16741@motorola.com>
Date: Mon, 4 Jan 2016 13:54:23 -0600
Subject: [PATCH 581/959] IKMMINTG-7328: arm/dts: msm8952: Adjust flash led
 currents

Increase combined warm/cool flash led strobe current to 1000mA.
Decrease torch led current to 75mA each.

Change-Id: I032103eb7dca60e152ea1baf412f9895a715e976
Signed-off-by: Andrew Lenart <w16741@motorola.com>
Reviewed-on: http://gerrit.mot.com/810473
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Reviewed-by: Satya Prasad Kottisa <fvgm76@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm-pmi8950.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/msm-pmi8950.dtsi b/arch/arm/boot/dts/qcom/msm-pmi8950.dtsi
index a18f13f1203..af8a7f12b19 100644
--- a/arch/arm/boot/dts/qcom/msm-pmi8950.dtsi
+++ b/arch/arm/boot/dts/qcom/msm-pmi8950.dtsi
@@ -557,7 +557,7 @@
 				qcom,max-current = <1000>;
 				qcom,duration = <1280>;
 				qcom,id = <0>;
-				qcom,current = <625>;
+				qcom,current = <1000>;
 			};
 
 			pmi8950_flash1: qcom,flash_1 {
@@ -568,7 +568,7 @@
 				qcom,max-current = <1000>;
 				qcom,duration = <1280>;
 				qcom,id = <1>;
-				qcom,current = <625>;
+				qcom,current = <1000>;
 			};
 
 			pmi8950_torch0: qcom,torch_0 {
@@ -578,7 +578,7 @@
 						"torch0_trigger";
 				qcom,max-current = <200>;
 				qcom,id = <0>;
-				qcom,current = <120>;
+				qcom,current = <75>;
 			};
 
 			pmi8950_torch1: qcom,torch_1 {
@@ -588,7 +588,7 @@
 						"torch1_trigger";
 				qcom,max-current = <200>;
 				qcom,id = <1>;
-				qcom,current = <120>;
+				qcom,current = <75>;
 			};
 
 			pmi8950_switch: qcom,switch {
@@ -599,7 +599,7 @@
 				qcom,max-current = <1000>;
 				qcom,duration = <1280>;
 				qcom,id = <2>;
-				qcom,current = <625>;
+				qcom,current = <1000>;
 				reg0 {
 					regulator-name = "pon_spare_reg";
 				};
-- 
2.11.0

